spec/forms/candidate_interface/english_gcse_grade_form_spec.rb

```diff
@@ -139,6 +139,15 @@
         expect(qualification.reload.constituent_grades).to eq({ 'english_double_award' => { 'grade' => 'CD', 'public_id' => next_available_public_id }, 'english_language' => { 'grade' => 'D', 'public_id' => next_available_public_id + 1 }, 'english_literature' => { 'grade' => 'D', 'public_id' => next_available_public_id + 2 } })
         expect(qualification.grade).to be_nil
       end
+
+      it 'creates a qualification with a default enic_reason value of not_needed' do
+        qualification = create(:gcse_qualification, subject: 'english', grade: nil)
```

Comment:
I assume this will get created with the default `enic_reason` 🤔

It's possible to create a qualification that has no reason using something like this https://github.com/thoughtbot/factory_bot/issues/578#issuecomment-572852808 to force the `enic_reason` to be `nil`